ssw/src/main/java/ssw/gui/DamageChart.java

```diff
@@ -37,10 +37,15 @@
 import javax.swing.JPanel;
 
 public class DamageChart extends JPanel {
+    /**
+     *
+     */
+    private static final long serialVersionUID = 1303342508543758261L;
+
     private int GridX = 1;
     private int GridY = 1;
-    private Vector charts = new Vector();
-    private Vector colors = new Vector();
+    private Vector<int[]> charts = new Vector<int[]>();
```

Comment:
I think you can do `private Vector<int[]> charts = new Vector<>();` instead, the type can be omitted in the constructor